dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java

```diff
@@ -1779,11 +1779,17 @@ private void validateMultiPartContent (final Map<String, Object> contentMap,
         return binaryFields.size() <= binaryFileSize? binaryFields: binaryFields.subList(0, binaryFields.size());
     }
 
+    private String getContentTypeInode (final Map<String, Object> contentMap, final User user, final List<File> binaryFiles) {
+
+        this.contentHelper.checkOrSetContentType(contentMap, user, binaryFiles);
+        return MapToContentletPopulator.INSTANCE.getContentTypeInode(contentMap);
+    }
+
     private void processFiles(final Map<String, Object> contentMap, final List<File> binaryFiles,
-                              final LinkedHashSet<String> argBinaryFields) throws DotDataException, DotSecurityException {
+                              final LinkedHashSet<String> argBinaryFields, final User user) throws DotDataException, DotSecurityException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using implementation types like 'LinkedHashSet'; use the interface instead](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5028307)